### DISCUSSION TITLE
Unable to install it

### DISCUSSION THREAD

**Discussion by szabizs** [spatie/ray#52] [Q&A]
I am unable to install the package.

PHP 7.4.3 
Composer 2.0.1
Laravel Framework 8.11.2

``` shell
Using version ^1.1 for spatie/laravel-ray
./composer.json has been updated
Running composer update spatie/laravel-ray
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires spatie/laravel-ray ^1.1 -> satisfiable by spatie/laravel-ray[1.1.0].
    - spatie/laravel-ray 1.1.0 requires illuminate/database ^7.0|^8.13 -> found illuminate/database[v7.0.0, ..., 7.x-dev, v8.13.0, ..., 8.x-dev] but it conflicts with another require.


Installation failed, reverting ./composer.json and ./composer.lock to their original content.

```

**Comment by freekmurze** [comment]
Our tests indicate that it is working as expected. Please try in a fresh Laravel app, or use commands like `composer why-not` to figure out why it is not working for your particular installation.

**Reply by szabizs** [reply]
Updated to latest version of Laravel and it is working, I believe that a requirement was too high.

```ssh
Package operations: 5 installs, 0 updates, 0 removals
  - Downloading symfony/stopwatch (v5.2.1)
  - Downloading spatie/macroable (1.0.1)
  - Downloading spatie/backtrace (1.0.0)
  - Downloading spatie/ray (1.2.0)
  - Downloading spatie/laravel-ray (1.1.0)
  - Installing symfony/stopwatch (v5.2.1): Extracting archive
  - Installing spatie/macroable (1.0.1): Extracting archive
  - Installing spatie/backtrace (1.0.0): Extracting archive
  - Installing spatie/ray (1.2.0): Extracting archive
  - Installing spatie/laravel-ray (1.1.0): Extracting archive
```

**Comment by haleemhosein** [comment]
> I am unable to install the package.
> 
> PHP 7.4.3
> Composer 2.0.1
> Laravel Framework 8.11.2
> 
> ```shell
> Using version ^1.1 for spatie/laravel-ray
> ./composer.json has been updated
> Running composer update spatie/laravel-ray
> Loading composer repositories with package information
> Updating dependencies
> Your requirements could not be resolved to an installable set of packages.
> 
>   Problem 1
>     - Root composer.json requires spatie/laravel-ray ^1.1 -> satisfiable by spatie/laravel-ray[1.1.0].
>     - spatie/laravel-ray 1.1.0 requires illuminate/database ^7.0|^8.13 -> found illuminate/database[v7.0.0, ..., 7.x-dev, v8.13.0, ..., 8.x-dev] but it conflicts with another require.
> 
> 
> Installation failed, reverting ./composer.json and ./composer.lock to their original content.
> ```

Yes I'm having this issue as well 😢 
Laravel Framework 5.7.29
PHP 7.4.13
Composer 2.0.8

**Reply by freekmurze** [reply]
You’ll need to be on Laravel 7 or higher

**Reply by haleemhosein** [reply]
Whoops, my bad. Didn't see the Requirements page 😅
